### PR TITLE
Make magic 8-ball more magical

### PR DESCRIPTION
**Pull request by xxv**
This adds accelerometer and touchpad support

### CODE REVIEW

File: magic8ball.py [hunk 1]
```diff
@@ -13,14 +13,98 @@
...
+    while keep_on[0]:
+        exit_button.read()
+        orientation = get_orientation(i2c)
+
+        if orientation and orientation != prev_orientation:
+            if orientation == 'upright':
+                show_message(urandom.choice(phrases))
+            elif orientation == 'prone':
+                clear_screen()
+            prev_orientation = orientation
```
Comment (xxv) [inline magic8ball.py:106]: BTW: this is in the correct location. I only want `prev_orientation` to change when `orientation` is not `None`.

**Comment by pdp7** [comment]
@xxv  I think this magic 8-ball app is effected by an issue with the OHS18 badge MicroPython firmware.

I rolled back to https://github.com/oshwabadge2018/micropython/commit/61cb6e3db593578c6532bd5ecd5af76174ed27
 
This is before the "Fixes the execution of py files with globals vars" by @mwelling in https://github.com/oshwabadge2018/micropython/commit/f3468954a67e0b6f689829ea90ca0d9b4bbe8b7e

With the older firmware, I have the same behavior as you.  This magic 8 ball app runs, and immediately exits without displaying anything on the e-paper.

The serial console shows:
> Launching <function execapp at 0x3ffe9b90>
> Running '/apps/magic8ball-48bd2c3.py'
> invalid menu index

I then checked out https://github.com/oshwabadge2018/micropython/commit/f3468954a67e0b6f689829ea90ca0d9b4bbe8b7e with fix by @mwelling and flashed that firmware.

The app now runs and display the magic 8-ball message on the e-paper:
> Launching <function execapp at 0x3ffe9b90>
> Running '/apps/magic8ball-48bd2c3.py'

Noticeably, the `invalid menu index` message is no longer printed to the console.

Could you try building and installing the current [MicroPython firmware](https://github.com/oshwabadge2018/micropython/)?

**Comment by acamilo** [comment]
invalid menu index is unrelated. That gets printed when you try to move off the end of the menu. For example, if you try selecting menu item 6 and only 5 options exist that will get printed.

**Comment by pdp7** [comment]
@xxv were you able to try the [latest micropython firmware](https://github.com/oshwabadge2018/ohs2018-badge-firmware)?

**Pull request merged**